workflow/admin.py

```diff
@@ -87,6 +87,13 @@ def formfield_for_foreignkey(self, db_field, request=None, **kwargs):
 
 class GaitIDInlineAdmin(admin.StackedInline):
     model = GaitID
+    fields = ['gaitid', 'donor', 'donor_dept']
+    readonly_fields = ['fund_codes',]
```

Comment:
On the Program page showing the `GaitIDInlineAdmin`, I do not see the fund_codes. I think the `fields` list needs `fund_codes` added to it as well.